webpack/components/extensions/HostDetails/Cards/ContentViewDetailsCard/ContentViewDetailsCard.js

```diff
@@ -77,7 +136,15 @@ const HostContentViewDetails = ({
                 justifyContent={{ default: 'justifyContentSpaceBetween' }}
               >
                 <FlexItem>
-                  <CardTitle>{__('Content view details')}</CardTitle>
+                  <CardTitle>
+                    <FormattedMessage
+                      id="cv-card-title"
+                      defaultMessage="{count, plural, =0 {Content view environments} one {Content view environment} other {Content view environments}}"
```

Comment:
Being picky here for our French customers but does this support translations.. 🇫🇷